1st-gen/tools/truncated/README.md

```diff
@@ -1,49 +1,241 @@
+
+```html demo
+<div
+    style="width: 200px; border: 1px solid var(--spectrum-gray-300); padding: 8px;"
+>
+    <sp-truncated success-message="Email copied successfully!">
+        iamaverylongusernametoshowthecustom.message.text@example.com
+    </sp-truncated>
+</div>
+```
+
+### Usage in patterns
 
-By default, tooltip text will be extracted from overflowing content. To provide your own overflow content, slot it into "overflow":
+#### With field labels
+
+`<sp-truncated>` is commonly used with form fields to display long values that might overflow:
+
+```html demo
+<sp-field-label for="email-field">Email address</sp-field-label>
+<div
+    style="width: 250px; border: 1px solid var(--spectrum-gray-300); padding: 8px; border-radius: 4px;"
+>
+    <sp-truncated>very.long.email.address@subdomain.example.com</sp-truncated>
+</div>
+```
+
+#### In popovers
+
+For use within overlays, ensure the popover has appropriate width constraints:
 
 ```html
-<sp-truncated placement="right">
-    This is the inline content
-    <span slot="overflow">
-        And this overflow content will go into the tooltip, on the right
-    </span>
-</sp-truncated>
+<sp-button id="user-trigger">User Info</sp-button>
+<sp-overlay trigger="user-trigger@click" placement="bottom">
+    <sp-popover style="width: 250px;">
+        <div style="padding: 16px;">
+            <sp-field-label>Username</sp-field-label>
+            <div style="width: 200px;">
+                <sp-truncated>
+                    very.long.username.that.exceeds.the.available.width
+                </sp-truncated>
+            </div>
+        </div>
+    </sp-popover>
+</sp-overlay>
 ```
+
+### Accessibility
+
+#### Keyboard navigation
+
+The `<sp-truncated>` component does not receive keyboard focus by itself, as it functions as inline content. However, the tooltip that appears on hover follows standard Spectrum tooltip accessibility patterns:
+
+- The tooltip appears on hover and focus
+- The tooltip is dismissed when the user moves away or presses the Escape key
+
+#### Click to copy
+
+When truncated text is clicked, the full text content is copied to the clipboard. This interaction provides a quick way for users to access the complete content.
+
+**Important accessibility note:** The click handler on the truncated text does not include keyboard event handlers (`/* eslint-disable lit-a11y/click-events-have-key-events */`). This is a known limitation. Users who rely on keyboard navigation will need to use the tooltip hover interaction to view the full content, but cannot copy it via keyboard alone.
+
+#### Screen readers
+
+Screen readers will announce the visible truncated text. When the tooltip appears on hover, screen readers will announce the full content if it differs from the visible text.
+
+For custom overflow content, ensure that the slotted content is semantically meaningful and properly structured for screen readers.
```

Comment:
- Since the `sp-truncated` does not receive keyboard focus, a keyboard user will _never_ be able to tooltip hover.
- Screenreader users--who are very often but not always keyboard users---will hear the full content without the tooltip.
- Not all keyboard users are screenreader users, so this is an accessibility bug.